pkg/event-manager/drivers/docker_backend.go

```diff
@@ -78,11 +79,14 @@ func (d *dockerBackend) getDriverSecrets(ctx context.Context, driver *entities.D
 	return secrets, nil
 }
 
-// Deploy deploys driver
-func (d *dockerBackend) Deploy(ctx context.Context, driver *entities.Driver) error {
-	log.Infof("Docker backend: deploying driver %v", driver.Name)
+// Expose deploys driver
+func (d *dockerBackend) Expose(ctx context.Context, driver *entities.Driver) error {
```

Comment:
Is this function necessary? Why does it only return nil? There should at least be a comment about why it is unimplemented and personally I think it should throw some sort of UnimplementedError particularly if this method is just here to implement an interface.

Reply:
Then I would recommend making that comment and having this return an an unimplemented error